docs/data/material/getting-started/templates/checkout/getCheckoutTheme.tsx

```diff
@@ -0,0 +1,816 @@
+              backgroundColor: alpha(brand[300], 0.1),
+              borderColor: brand[300],
+              color: brand[500],
+              '&:hover': {
+                backgroundColor: alpha(brand[300], 0.3),
+                borderColor: brand[200],
+              },
+            }),
+            ...(ownerState.variant === 'text' && {
+              color: brand[500],
+              '&:hover': {
+                backgroundColor: alpha(brand[300], 0.3),
+                borderColor: brand[200],
+              },
+            }),
+            ...(theme.palette.mode === 'dark' && {
+              ...(ownerState.variant === 'contained' &&
+                ownerState.color === 'primary' && {
+                  border: `1px solid ${brand[600]}`,
+                  '&:hover': {
+                    background: brand[600],
+                    backgroundImage: 'none',
+                    boxShadow: `0 0 0 1px  ${alpha(brand[700], 0.5)}`,
+                  },
+                }),
+              ...(ownerState.variant === 'outlined' && {
+                backgroundColor: alpha(brand[600], 0.1),
+                borderColor: brand[700],
+                color: brand[300],
+                '&:hover': {
+                  backgroundColor: alpha(brand[600], 0.3),
+                  borderColor: brand[700],
+                },
+              }),
+              ...(ownerState.variant === 'text' && {
+                color: brand[300],
+                '&:hover': {
+                  backgroundColor: alpha(brand[600], 0.3),
+                  borderColor: brand[700],
+                },
+              }),
+            }),
+          }),
+        },
+      },
+      MuiCard: {
+        styleOverrides: {
+          root: ({ theme, ownerState }) => ({
+            backgroundColor: gray[50],
+            borderRadius: 10,
+            outline: `1px solid ${alpha(gray[200], 0.8)}`,
+            boxShadow: 'none',
+            transition: 'background-color, border, 80ms ease',
+            ...(ownerState.variant === 'outlined' && {
+              boxSizing: 'border-box',
+              background: `linear-gradient(to bottom, #FFF, ${gray[50]})`,
+              '&:hover': {
+                borderColor: brand[300],
+                boxShadow: `0 0 24px ${brand[100]}`,
+              },
+            }),
+            ...(theme.palette.mode === 'dark' && {
+              backgroundColor: alpha(gray[800], 0.6),
+              outline: `1px solid ${alpha(gray[700], 0.3)}`,
+              ...(ownerState.variant === 'outlined' && {
+                boxSizing: 'border-box',
+                background: `linear-gradient(to bottom, ${gray[900]}, ${alpha(
+                  gray[800],
+                  0.5,
+                )})`,
+                '&:hover': {
+                  borderColor: brand[700],
+                  boxShadow: `0 0 24px ${brand[800]}`,
+                },
+              }),
+            }),
+          }),
+        },
+      },
+      MuiChip: {
```

Comment:
Do we use a chip in this template? 🤔

Reply:
Actually no. However, this is something that we have to decide. Currently, we have one theme file for each template. So, we have to decide if we're going to have the content individualized by templates, or if, even though separated, the theme files will share the same content.

I'm leaning towards the latter, because users can have a complete stylesheet regardless of the template they choose, meaning that if they, for some reason, add a chip to the checkout template in their project, the styles would already be there.

Reply:
I'd have the theme file individualized by the template, containing code only related to the components used in the template. That way, we make each template encapsulated and not dependent on any other, even though they're being designed as if they were all part of the same app (and thus the same visual language). I wouldn't include theme customizations for components that weren't needed, as that's a much bigger scope (which we can definitely tackle in the future, where folks could "download" fully loaded themes, which may or may not come with templates).

If we ever want to make multiple templates "clonable"/"downloadable" at once, where they all feel like a cohesive/unified app, there might be a different way to do it.